test/module/irohad/torii/query_service_test.cpp

```diff
@@ -88,6 +88,7 @@ TEST_F(QueryServiceTest, InvalidWhenDuplicateHash) {
 
   init();
 
+  //TODO luckychess 03.02.2018 is this OK?
```

Comment:
This comment looks not Ok. Better to explain the concerns that you found and make a task that solves it.